tsl/src/compression/create.c

```diff
@@ -589,7 +590,8 @@ create_compression_table(Oid owner, CompressColInfo *compress_cols)
 	create->constraints = NIL;
 	create->options = NULL;
 	create->oncommit = ONCOMMIT_NOOP;
-	create->tablespacename = NULL;
+	create->tablespacename =
+		(!OidIsValid(tablespace_oid)) ? NULL : get_tablespace_name(tablespace_oid);
```

Comment:
I don't think we need Elvis here - it must be set correctly

![Elvis](https://media.giphy.com/media/21jsTJ0SyBehCaiRZC/giphy.gif)

Reply:
In my opinion, it would be good to add an assert in case of `OidIsValid(tablespace_oid)` and `get_tablespace_name` returns NULL.

Reply:
I have added the assertion